packages/usa-tooltip/src/index.js

```diff
@@ -317,6 +317,7 @@ const setUpAttributes = (tooltipTrigger) => {
   const tooltipContent = tooltipTrigger.getAttribute("title");
   const wrapper = document.createElement("span");
   const tooltipBody = document.createElement("span");
+  const ariaSpan = document.createElement("span");
```

Comment:
**Question:** Can we add `tooltip: ` to the existing span instead of creating a new `span` element?
    
 I notice while navigating by item that it identifies the new span as an empty group and can be confusing / verbose for SR users:

[Kapture 2024-11-26 at 17.19.42.webm](https://github.com/user-attachments/assets/f19c5af6-6c3b-4310-920b-9fc0c435d139)

This would also help us keep the output markup a bit leaner!